addons/mail/static/tests/discuss/call/call.test.js

```diff
@@ -49,6 +50,20 @@ test("basic rendering", async () => {
     await contains("[title='Enter Full Screen']");
 });
 
+test("keep the `more` popover active when hovering it", async () => {
+    mockGetMedia();
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    await start();
+    await openDiscuss(channelId);
+    await click("[title='Start a Call']");
+    await contains(".o-discuss-Call");
+    await contains(".o-discuss-CallActionList");
+    await click("[title='More']");
+    hover(queryFirst(".o-dropdown-item[title='Enter Full Screen']"));
+    await contains(".o-dropdown-item[title='Enter Full Screen']");
```

Comment:
the `await click("[title='More']")` doesn't necessarily immediately show the dropdown, so better have `contains(".o-dropdown-item")` just after,

Then you can `hover(queryFirst())` it, and the follow-up `contains()` should need a `tick()` or some other `contains()` to give a bit more chance that the kept dropdown is result of hover.
Indeed, due to how `contains()` await until it matches, here nothing tells that the `contains()` is resolved just before the handling of hover, and the hover actually removes it eventually...